### ISSUE TITLE
cross validation error if first and last trial are both in the same test set

### ISSUE THREAD

**Issue by jzaremba** [nicholas-roy/psytrack#5]
If the first and last trial are both in the same test fold, then sum(missing_trials) is one less than len(test_trials). This leads to logli and gw from Kfold_crossVal_check to be one element short.

The actual error I am getting is in makeWeightPlot at `X = np.array([i['gw'] for i in prediction]).flatten()`. This can be avoided by switching to np.hstack, but then I think that may mess up the indexing of the array.

More generally, whenever 0 is in the test fold, it wraps around and assigns it's 'missing trial' to the last trial (instead of no trial).

I'm not actually sure how the cross validated prediction should be made for the very first trial, do you have any suggestions?

Thanks for the great package.

**Comment by nicholas-roy** [comment]
Thanks Jeff! I'll fix this is the next release (~1 week)!

The 0 trial in the test set does seem to be an issue — naively I would just
have it take the value of the smallest trial in the training set, but it
might just be easiest to explicitly require trial 0 to be in the training
set.

**Comment by jzaremba** [comment]
Great, thanks.

**Comment by nicholas-roy** [comment]
A bit late, but this is fixed in the most recent version 1.2.0 — thanks again for the catch!
